MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import WebKit
+
+protocol MiniAppCallbackProtocol: AnyObject {
+    func didRecieveScriptMessageResponse(messageId: String, response: String)
+    func didRecieveScriptMessageError(messageId: String, errorMessage: String)
+}
+
+internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
+
+    weak var delegate: MiniAppCallbackProtocol?
+    var hostAppMessageDelegate: MiniAppMessageProtocol?
```

Comment:
Does this delegate need to be weak?

Reply:
yes it preferably does, this avoids reference cycles

Reply:
Done 👍